bitcash/network/services.py

```diff
@@ -204,6 +206,22 @@ def get_unspent_testnet(cls, address):
             for tx in r.json()['utxos']
         ]
 
+    @classmethod
+    def get_raw_transaction(cls, txid):
+        r = requests.get(cls.MAIN_RAW_API.format(txid), timeout=DEFAULT_TIMEOUT)
+        if r.status_code != 200:  # pragma: no cover
```

Comment:
I would like to replace lines 212 -> 213 with `r.raise_for_status()  # pragma: no cover` 😀